types/loader.d.ts

```diff
@@ -0,0 +1,138 @@
+declare namespace PIXI {
```

Comment:
Is this needed? Couldn't you just have the script load the typings in `node_modules/resource-loader/typings/resource-loader.d.ts`?

Reply:
resource-loader typings depend on mini-signals which are not exposed in pixi. @mat's Runner is not exposed too. When I tried to expose signals my PR was rejected.

Reply:
Pixi's renderer is just aliased resource-loader, signals are actually exposed.

You can do this:

```
PIXI.loader.onComplete.add()
```